code/modules/projectiles/projectile/bullets.dm

```diff
@@ -101,18 +101,6 @@
 	if(H.getStaminaLoss() >= 60)
 		H.KnockDown(8 SECONDS)
 
-/obj/item/projectile/bullet/pellet/weak
-	tile_dropoff = 0.55		//Come on it does 6 damage don't be like that.
-	damage = 6
-
-/obj/item/projectile/bullet/pellet/weak/New()
-	range = rand(1, 8)
-	..()
-
-/obj/item/projectile/bullet/pellet/weak/on_range()
-	do_sparks(1, 1, src)
-	..()
-
 /obj/item/projectile/bullet/pellet/overload
 	damage = 3
 
```

Comment:
Does anything else keep these overloaded pellets?

Reply:
Oh I missed that one, thanks